Laravel.gitignore

```diff
@@ -8,9 +8,14 @@ app/storage/
 # Laravel 5 & Lumen specific
 public/storage
 storage/*.key
+Homestead.json
+Homestead.yaml
 .env.*.php
 .env.php
 .env
 
+# Logs
```

Comment:
Could you make this comment clearer that it's related to the use of Larvel Elixir?